x42-plugins/riscv64.patch

```diff
@@ -0,0 +1,32 @@
+diff --git PKGBUILD PKGBUILD
+index f246ee5d..7147ffc6 100644
+--- PKGBUILD
++++ PKGBUILD
+@@ -19,20 +19,24 @@ optdepends=('jack: for standalone applications'
+ source=("http://gareus.org/misc/${pkgname}/${pkgname}-${pkgver}.tar.xz"
+         "${pkgname}.png::https://x42-plugins.com/x42/static/img/x42.png"
+         "${pkgname}.directory"
+-        "${pkgname}.menu")
++        "${pkgname}.menu"
++        "fix-compile.patch")
```

Comment:
Please use a more descriptive name for the patch, for example `disable-sse`

Reply:
Fixed.